tests-integration/composer.py

```diff
@@ -0,0 +1,50 @@
+import os
+from airflow import DAG
+from datetime import datetime
+from kubernetes.client import models as k8s
+from airflow.models import Variable
+from dataverk_airflow import python_operator, notebook_operator, quarto_operator
+
+# Env variable for GCP bucket is automatically set for composer instances
+# For the test we must set it manually
+os.environ["GCS_BUCKET"] = "dataverk-airflow-tests"
```

Comment:
Er ikke bedre å bruke `@mock.patch.dict(os.environ,...`, som vi gjør i k8s operator testen.

Reply:
Det kreves at det er en klasse for å kunne bruke den decoratoren.